src/components/Tables/Table.tsx

```diff
@@ -618,6 +620,28 @@ const TableStyling = styled.div`
     & .rdt_TableRow:nth-child(odd) {
         background-color: ${(props) => props.theme.main.background[0]};
     }
+
+    ${(props) =>
+        props.rowBreakPoints
+            ?.slice()[1]
+            .map(
+                (lightGreenBreakPoint) =>
+                    `& .rdt_TableRow:nth-child(${lightGreenBreakPoint + 1}) {
+                border-top: solid #77bda0;
+            }`
+            )
+            .join()}
+
+    ${(props) =>
+        props.rowBreakPoints
+            ?.slice()[0]
+            .map(
+                (darkGreenBreakPoint) =>
```

Comment:
We also want this code to be able to deal with an arbitrary number of columns that we're using to find the breakpoints by, and we don't want the component to be the one to deal with the colours/significance of each such column, but the ParcelsTable file, so that we can use the breakpoints feature in other tables as flexibly as possible in the future if we want to. So maybe the data structure that I was talking about should also store the colour of the dividing line.